engines/jsx.js

```diff
@@ -7,14 +7,14 @@ if (!globalThis.React) {
 import Module from "./module.js";
 
 export default class Jsx extends Module {
-  render(content, data) {
+  async render(content, data) {
     if (!data.children && data.content) {
       data.children = React.createElement("div", {
         dangerouslySetInnerHTML: { __html: data.content },
       });
     }
 
-    const element = super.render(content, data);
+    const element = await Promise.resolve(super.render(content, data));
```

Comment:
You don't need to create a Promise. `await` can handle any returned value, Promise or not.

```suggestion
    const element = await super.render(content, data);
```

Reply:
Thanks for the feedback!

I wasn't 100% sure that that was always the case, but I've incorporated your suggestion in the latest commit.